plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -387,7 +473,7 @@ public void killContainer(String container, int signal) throws IOException {
      * @throws IOException
      */
     public void killContainer(String container) throws IOException {
-        killContainer(container, 9);
+        killContainer(KillContainerParams.from(container));
```

Comment:
Because it is default signal according to docker API: https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#kill-a-container